workspace/apps/pidp/src/app/features/faq/pages/mfa-setup/mfa-setup.page.html

```diff
@@ -0,0 +1,257 @@
+<ui-page mode="full">
+  <ui-page-header
+    >BCProvider AD Multi-factor Authentication (MFA) Setup</ui-page-header
+  >
+
+  <ui-page-section>
+    <ui-page-section-subheader
+      icon="question_answer"
+      heading="Setting up the Microsoft Authenticator">
+    </ui-page-section-subheader>
+  </ui-page-section>
+  <ui-page-section>
+    <ol>
+      <li>Go to bchealthprovider.ca</li>
+      <li class="mfa-section">
+        Enter your bcprovider.ca login information
+        <picture class="image">
+          <source
+            type="image/webp"
+            srcset="/assets/images/mfa-setup/image1.webp" />
+          <source
+            type="image/jpg"
+            srcset="/assets/images/mfa-setup/image1.jpg" />
+          <img title="signin" loading="lazy" aria-labelledby="" />
+        </picture>
+      </li>
+      <li class="mfa-section">
+        Enter your password
+        <picture class="image">
+          <source
+            type="image/webp"
+            srcset="/assets/images/mfa-setup/image2.webp" />
+          <source
+            type="image/jpg"
+            srcset="/assets/images/mfa-setup/image2.jpg" />
+          <img title="password" loading="lazy" aria-labelledby="" />
+        </picture>
+      </li>
+      <li class="mfa-section">
+        You will be prompted for more information on the first login.
+      </li>
+      <li class="mfa-section">
+        On your iOS/Android mobile device, you will need to go to the respective
+        app store and install the
+        <a
+          uiAnchor
+          [attr.href]="'https://www.microsoft.com/en-ca/security/mobile-authenticator-app'"
```

Comment:
No need to add a square bracket when you pass a string as a parameter.
```suggestion
          href="https://www.microsoft.com/en-ca/security/mobile-authenticator-app"
```